mongodb-plugins/src/main/java/co/cask/hydrator/plugin/batch/sink/MongoDBBatchSink.java

```diff
@@ -57,7 +59,13 @@ public MongoDBBatchSink(MongoDBSinkConfig config) {
 
   @Override
   public void prepareRun(BatchSinkContext context) throws Exception {
-    context.addOutput(Output.of(config.referenceName, new MongoDBOutputFormatProvider(config)));
+    Configuration conf = new Configuration();
+    String path = conf.get(
+      "mapreduce.task.tmp.dir",
```

Comment:
now im even more confused, doesn't this mean if any of these setting exist (don't they always exist?), then the path will be the same for all sinks?